lms/templates/instructor/edx_ace/accountcreationandenrollment/email/body.html

```diff
@@ -31,6 +31,13 @@ <h1>
                 <br />
             </p>
 
+            <p style="color: rgba(0,0,0,.75);">
+		    {% blocktrans with start_link="<a href='{{ course_url }}'" end_link="</a>" trimmed %}
```

Comment:
@gr4yscale It seems there's a `>` missing.

You should also update translations, since I guess there are new ones.

Reply:
Oof, thanks for that catch.